utils/spdx/src/test/kotlin/SpdxExpressionTest.kt

```diff
@@ -472,6 +472,13 @@ class SpdxExpressionTest : WordSpec() {
                 spdxExpression.isValidChoice("b AND c AND e".toSpdx()) shouldBe true
             }
 
+            "return true if a choice is valid in a mixed expression" {
+                val spdxExpression = "a AND (a OR b)".toSpdx()
+
+                spdxExpression.isValidChoice("a".toSpdx()) shouldBe true
+                spdxExpression.isValidChoice("a AND b".toSpdx()) shouldBe true
```

Comment:
Should this better be made a `"validChoices()"` test (using `containExactlyInAnyOrder`) to also verify these are the *only* valid choices?

Reply:
You also need to move the test to the `"validChoices()" should` section.